packages/nodes-base/nodes/Aws/Cognito/descriptions/user/User.resource.ts

```diff
@@ -0,0 +1,229 @@
+import type { INodeProperties } from 'n8n-workflow';
+
+import * as addToGroup from './addToGroup.operation';
+import * as create from './create.operation';
+import * as del from './delete.operation';
```

Comment:
please use the same naming convention to keep consistency
```suggestion
import * as delete from './delete.operation';
```